money_calculation.py

```diff
@@ -0,0 +1,12 @@
+price = str(input('Input price, separate dollars from cents with "." : '))
+num_of_items = int(input('Input number of items you want to buy: '))
+dollars = int(price.split('.')[0])
+cents = int(price.split('.')[1])
+print('One item price is {0} dollars {1} cents'.format(dollars, cents))
```

Comment:
In description wasn't mentioned this print

Reply:
Solved.